src/backend/common/ConfigManager.ts

```diff
@@ -123,6 +123,20 @@ const migrations: [string, (store: Record<string, unknown>) => unknown][] = [
       return config;
     },
   ],
+  [
+    ">=0.16",
```

Comment:
問題ないけどなんとなく今のバージョン合わせ
```suggestion
    ">=0.17",
```